android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/notifications/ScopedServerRegistrationModule.java

```diff
@@ -8,14 +8,16 @@
 
 import expo.modules.notifications.serverregistration.ServerRegistrationModule;
 import host.exp.exponent.di.NativeModuleDepsProvider;
+import host.exp.exponent.kernel.ExperienceId;
 import host.exp.exponent.storage.ExponentSharedPreferences;
 
 public class ScopedServerRegistrationModule extends ServerRegistrationModule {
   @Inject
   ExponentSharedPreferences mExponentSharedPreferences;
 
-  public ScopedServerRegistrationModule(Context context) {
+  public ScopedServerRegistrationModule(Context context, ExperienceId experienceId) {
```

Comment:
I think this means we need to update `detachedModuleRegistryAdapter` and pass in `experienceId` here- https://github.com/expo/expo/blob/%40sjchmiela/expo-notifications-managed-identifier/android/expoview/src/main/java/host/exp/exponent/experience/DetachedModuleRegistryAdapter.java#L63